code/_onclick/click.dm

```diff
@@ -511,9 +511,10 @@
 		var/mob/living/carbon/human/H = usr
 		H.swap_hand()
 	else
-		var/turf/T = params2turf(modifiers["screen-loc"], get_turf(usr.client?.eye ? usr.client.eye : usr), usr.client)
+		var/turf/T = params2turf(modifiers["screen-loc"], get_turf(usr.client ? usr.client.eye : usr), usr.client)
 		params += "&catcher=1"
-		T?.Click(location, control, params)
+		if(T)
```

Comment:
This check doesn't affect behavior. But damn profiling showed this to be more efficient regarding procs...

Reply:
Yep technically I could atomize this, but that to me felt like grabbing PRs since this is a 4 line PR by itself

Reply:
Nah, no need.